src/gpo/assets/PowerToys.admx

```diff
@@ -326,6 +327,16 @@
         <decimal value="0" />
       </disabledValue>
     </policy>
+    <policy name="ConfigureEnabledUtilityNew" class="Both" displayName="$(string.ConfigureEnabledUtilityNew)" explainText="$(string.ConfigureEnabledUtilityDescription)" key="Software\Policies\PowerToys" valueName="ConfigureEnabledUtilityNew">
```

Comment:
I know it's early state. And maybe you don't like to use the character `+`. But only `new` can be confusing and interpreted as adjective. So we should use `NewPlus` as utility name.

Reply:
Thank you @htcfreek!  I've changed it to ConfigureEnabledUtilityNewPlus